tests/visual.spec.ts

```diff
@@ -138,15 +144,6 @@ async function setEyesSettings() {
     ],
   })
 
-  const time = new Date().toISOString()
-  const user = username() || 'unknown-user'
-  const batchName = process.env.APPLITOOLS_BATCH_NAME
-    ? `${process.env.APPLITOOLS_BATCH_NAME}visual-tests`
-    : `${user}-visual-tests-${time}`
-  const batchId = process.env.SHA || `${user}-${time}`
-
-  eyes.setBatch({ name: batchName, id: batchId })
```

Comment:
why not having the user name in the batch name? that's nice (it affects the name in the left menu)